web/viewer.css

```diff
@@ -322,7 +325,6 @@ body {
 :is(.toolbar, .editorParamsToolbar, .findbar, #sidebarContainer)
   :is(input, button, select),
 .secondaryToolbar :is(input, button, a, select) {
-  outline: none;
```

Comment:
This affects much more than HCM-mode, note how e.g. the zoom-dropdown now has an unexpected/inconsistent outline when tabbing through the toolbar buttons.
The same "issue" also applies to the entries in the various editor-toolbars.